module/Makefile.bsd

```diff
@@ -68,6 +68,10 @@ CFLAGS+= -DBITS_PER_LONG=32
 CFLAGS+= -DBITS_PER_LONG=64
 .endif
 
+.if ${MACHINE_CPUARCH} != "mips" && ${MACHINE_ARCH} != "arm"
+CFLAGS+= -DHAVE_LARGE_STACKS=1
+.endif
+
```

Comment:
But here I am at loss: I haven't looked on the build process, but it looks weird to me first detect the stack size in configure, but then hardcode it.

Reply:
This is a non-autotools build and is self-sufficient BSDmakefile.  It is done for completeness.

Reply:
Just to confirm i386 and powerpc use 16k stacks by default as well?

I'm also not sure the autoconf change really buys us much if we're just going to hardcode it here, but I'm not opposed to it.

Reply:
Yes, both of those use 4 stack pages on FreeBSD.

There seems to be a misunderstanding of what this file even is, which is orthogonal to this changeset.  It's a self-contained BSDmakefile to build the kmod and does not tie in to autotools at all, it uses the bsd.mk framework.  Whether it is unnecessary cruft maybe is something you are both encircling.

Reply:
What I was driving at is the `Makefile.bsd` file is used by the normal `make modules` target in this repository as well.  So there shouldn't be a need to add it to the autoconf bits.

Reply:
Ok which side should I remove it from?

Reply:
That looks better to me and I believe it should be sufficient.  Thanks.

Reply:
Ok I see the tie-in now and this will more closely match the downstreaming.